studio/components/interfaces/Settings/Logs/LogsErrorRenderers/ResourcesExceededErrorRenderer.tsx

```diff
@@ -19,7 +19,6 @@ const ResourcesExceededErrorRenderer: React.FC<ErrorRendererProps> = ({ error, i
       type="default"
       chevronAlign="left"
       size="small"
-      bordered={false}
```

Comment:
This bordered prop wasn't being used in the component, so removing it from here.